pkg/cli/cli.go

```diff
@@ -826,11 +824,15 @@ func runCreateCVDCommand(c *cobra.Command, args []string, flags *CreateCVDFlags,
 			}
 		}
 	}
+	hostSrvURL, err := srvClient.HostServiceURL(flags.CreateCVDOpts.Host)
+	if err != nil {
+		return fmt.Errorf("failed getting host service url: %w", err)
+	}
 	hosts := []*RemoteHost{
 		{
-			ServiceRootEndpoint: srvClient.RootURI(),
-			Name:                flags.CreateCVDOpts.Host,
-			CVDs:                cvds,
+			ServiceURL: hostSrvURL,
```

Comment:
I prefer `HostServiceURL` rather than `ServiceURL` in perspective of code readers. AFAIU, we're using the term `ServiceURL` as the address of Cloud Orchestration widely.

Reply:
Good point, I thought about it as well, decided to stick with `ServiceURL` as the `Host` prefix would be redundant as it's a field of the `RemoteHost` struct. 

````
host := RemoteHost{...}` 
host.ServiceURL // I think it's clear in this case this is the url of the host orchestrator api
```

Reply:
Understood. It sounds fair to me.